supply-chain/config.toml

```diff
@@ -19,6 +19,9 @@ url = "https://raw.githubusercontent.com/divviup/libprio-rs/main/supply-chain/au
 [imports.mozilla]
 url = "https://raw.githubusercontent.com/mozilla/supply-chain/main/audits.toml"
 
+[imports.zcash]
+url = "https://raw.githubusercontent.com/zcash/rust-ecosystem/main/supply-chain/audits.toml"
```

Comment:
uhm. Nope, let's please not do that. So far Mozilla doesn't import zcash's audits, so unless I hear otherwise Glean won't do it either.
I can take care of `cargo-vet` for this.

Reply:
Ah, I was doing that and saw the suggestion. Will roll back. 0.4.19 is totally fine while I need to take a look at 0.4.20